source/modulo_components/modulo_components/component_interface.py

```diff
@@ -32,6 +32,7 @@ class ComponentInterface(Node):
         predicates (dict(Parameters(bool))): map of predicates added to the Component.
         predicate_publishers (dict(rclpy.Publisher(Bool))): map of publishers associated to each predicate.
         parameter_dict: dict of parameters
+        periodic_callbacks: dict of periodic callback functions
```

Comment:
Can you add `(dict(Callable))` and `dict(Parameters)` the line above just to make it consistent? (Sorry I can't make suggestions it seems.

Reply:
I actually don't fully agree, and I'd rather remove the type hints in the docstring because I have them now very nice when I define the attribute (check a few lines further down). Also `dict(Parameters)` is not technically correct because it can be a parameter or it can be the name of an attribute that refers to the parameter...
I think suggestions don't work if you review by commit

Reply:
I added it for now, we have to figure out how we can do this properly later